src/util.rs

```diff
@@ -85,6 +86,17 @@ pub fn format_count(num: i32) -> String {
     }
 }
 
+/// Find all emojis in a String
+pub fn find_emojis(value: impl AsRef<str>) -> Vec<EmojiIdentifier> {
+    lazy_static::lazy_static! {
+        static ref FIND_EMOJI : regex::Regex = regex::Regex::new(r"<:\s*?(\w\w+?):\s*?([0-9]\w+?)>|<a:\s*?([a-z]\w+?):\s*?([0-9]\w+?)>").unwrap();
```

Comment:
It would be great if we could somehow get rid of the capture groups here (or at least minimize them)... But dunno how feasible that is.
does it _have_ to be this complex?

Reply:
It's so you can get the flake,the id, the name, and for both types of emojis

Reply:
`<a?:[0-9a-zA-Z_]{2,32}:[0-9]{18,}>` should work.
If you add the capture groups here, then you should also be able to yeet the `serenity::utils::parse_emoji` call and replace it with just the values from the capture groups. alternatively, if you _don't_ add the capture groups, then you should be able to use the parse_emoji function from serenity still - currently, you have capture groups that you don't use, which is needlessly imperformant